docs/ReduceBinarySize.md

```diff
@@ -5,6 +5,7 @@ To reduce compiled binary size, two options are available:
 - --include_ops_by_file=<path to a file\>
 
 The options empower building to comment out operators listed in execution provider(s), thereby downsizing the output.
+Note that please build with --skip_tests in case excluded ops cause test failure.
 
```

Comment:
I don't think you need to say this in two places so close together.

I'd suggest it's a bit louder though. 'You MUST build with --skip_tests'